pkg/globalnet/controllers/types.go

```diff
@@ -129,3 +130,12 @@ type nodeController struct {
 	nodeName string
 	nodes    dynamic.ResourceInterface
 }
+
+type podController struct {
+	*baseSyncerController
+	pods         dynamic.NamespaceableResourceInterface
+	scheme       *runtime.Scheme
+	svcName      string
+	namespace    string
+	ingressIPMap map[string]bool
```

Comment:
Looks like this can be a set.

```suggestion
	ingressIPs        stringset.Interface
```